views/player_mdk.html

```diff
@@ -32,7 +32,9 @@
 {{! Change the mustache brackets to ERB style so that angular code won't get messed up }}
 {{=<% %>=}}
 <div class="widget" ng-app="materia">
-	<section class="widget" ng-controller="playerCtrl" ng-init="inst_id = 'demo'" ng-class="{ preview: isPreview }">
+	<section class="widget" ng-controller="playerCtrl"
+		ng-init="inst_id = <% instance %>"
```

Comment:
```suggestion
		ng-init="inst_id = '<% instance %>'"
```

This line needs quotes around it to deal with "demo".